packages/console/src/pages/SignInExperience/components/SignUpAndSignInChangePreview/SignUpAndSignInDiffSection/index.tsx

```diff
@@ -12,7 +14,11 @@ type Props = {
 
 const SignUpAndSignInDiffSection = ({ before, after, isAfter = false }: Props) => (
   <>
-    <SignUpDiffSection before={before.signUp} after={after.signUp} isAfter={isAfter} />
+    <SignUpDiffSection
+      before={signInExperienceParser.toLocalSignUp(before.signUp)}
+      after={signInExperienceParser.toLocalSignUp(after.signUp)}
```

Comment:
These are a bit confusing. Can we pass `before` and `after` into this component and call the parser inside?